Innowise.Clinic.Auth.IntegrationTesting/TestHelper.cs

```diff
@@ -0,0 +1,66 @@
+using System;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using Innowise.Clinic.Auth.Constants;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+
+namespace Innowise.Clinic.Auth.IntegrationTesting;
+
+public static class TestHelper
+{
+    
+    internal const string SignUpEndpointUri =
+        ControllerRoutes.AuthenticationControllerRoute + "/" + EndpointRoutes.SignUpEndpointRoute;
+
+    internal const string RefreshTokenEndpointUri =
+        ControllerRoutes.AuthenticationControllerRoute + "/" + EndpointRoutes.RefreshTokenEndpointRoute;
+
+    internal const string SignInEndpointUri = ControllerRoutes.AuthenticationControllerRoute + "/" +
+                                              EndpointRoutes.SignInEndpointRoute;
+
+    internal const string SignOutEndpointUri = ControllerRoutes.AuthenticationControllerRoute + "/" +
+                                               EndpointRoutes.SignOutEndpointRoute;
+
+    
+    internal static int _uniqueNumber = 0;
+
+    internal static int UniqueNumber => _uniqueNumber++;
```

Comment:
Constants in tests is bad practice because if we change value of constant tests will not fail but all frontends will.